experiments/modern/src/initialize.js

```diff
@@ -1,8 +1,31 @@
 import * as Utils from "./utils";
-import JSZip from "jszip";
+
+function splitValues(s) {
+  const retArr = [];
+  const arr = s.split(",");
+  for (let i = 0; i < arr.length; i++) {
+    retArr.push(arr[i]);
+  }
+  return arr.map((v) => parseFloat(v));
+}
+
+async function loadImage(imgUrl) {
+  return await new Promise(resolve => {
+    const img = new Image();
+    img.addEventListener("load", function() {
+      resolve(img);
+    });
+    img.src = imgUrl;
+  });
+}
+
+function uuid () {
+  return '_' + Math.random().toString(36).substr(2, 9);
+}
```

Comment:
This could probably just be:

```
let id = 0;
function getId() {
  return id++;
}

Reply:
haha, had intended on coming back to that with something less hacky ..... but yea, really no reason I can just increment! haha